data/raw/datapackage.json

```
@@ -33,35 +14,35 @@
     "resources": [
         {
             "name": "morfometria_albatros-laysan_guadalupe",
-            "description": "Morfometría de albatros de Laysan en Isla Guadalupe (2015-2016)",
             "encoding": "utf-8",
             "format": "csv",
             "mediatype": "text/csv",
             "path": "laysan_albatross_morphometry_guadalupe.csv",
             "profile": "tabular-data-resource",
-            "source": "Medidas in situ",
-            "title": "Laysan Albatross morphometry from Guadalupe Island",
-            "titulo": "Morfometría de Albatros de Laysan en Isla Guadalupe",
+            "source": "Albatros de Laysan anidados en Isla Guadalupe durante las temporadas de 2015 a 2018",
+            "title": "Laysan albatross morphometry from Guadalupe Island",
+            "titulo": "Morfometría de albatros de Laysan en Isla Guadalupe",
+            "description": "Esta base de datos contiene registros de 11 variables morfométricas de 135 individuos de albatros de Laysan en Isla Guadalupe. Los muestreos se realizaron durante las temporadas de 2015 a 2018.",
             "schema": {
                 "fields": [
                     {
-                        "name": "Temporada",
+                        "name": "temporada",
                         "axis": "T",
-                        "long_name": "Year",
-                        "nombre_largo": "Año",
-                        "standard_name": "date",
+                        "long_name": "Season",
+                        "nombre_largo": "Temporada",
                         "type": "integer",
-                        "description": "Año de periodo de monitoreo durante temporada de reproducción de Albatros Laysan",
+                        "description": "Año en el que se realizó el monitoreo durante la temporada de reproducción de albatros de Laysan.",
                         "constraints": {
-                            "minumum": 2015, "maximum": 2018
+                            "minumum": 2015,
+                            "maximum": 2018
                         }
                     },
                     {
                         "name": "subcolonia",
                         "long_name": "Subcolony",
                         "nombre_largo": "Subcolonia",
                         "type": "string",
-                        "description": "Colonia de distrubución interna de Isla Guadalupe",
+                        "description": "Colonia de distrubución interna de Isla Guadalupe.",

```

En general, la palabra isla la dejamos con minúsculas.
```suggestion
                        "description": "Colonia de distrubución interna de isla Guadalupe.",

```

---

Holi @andrea-sanchez  en los constraints de enum, ¿puedes ordenar los campos alfabéticamente ✏️  por fis?

```
                        "enum": [
                                "Morro Prieto",
                                "Punta Sur",
                                "Zapato",
                            ]

```